index.html

```diff
@@ -56,7 +56,7 @@
         <div class="row align-items-center">
           
           <div class="col-6 col-xl-2">
-          <h1 class="mb-0 site-logo"><a href="index.html" class="h2 mb-0"><img class="classLogo" src="./images/robocin-04.png" height="75px" width="245px"><span class="img-primary"></span> </a></h1>
+          <h1 class="mb-0 site-logo"><a href="index.html" class="h2 mb-0"><img class="classLogo" src="./images/logo.svg" height="75px" width="245px" style="height: 50px !important; padding-top: 0px !important; margin-top: 10px !important; margin-bottom: 10px !important;"><span class="img-primary"></span> </a></h1>

```

Comment:
Use logo in horizontal version. Available at [https://drive.google.com/file/d/1iE6Fr6l4l2w7HZ04uKKG-_p-HOQeRpON/view?usp=sharing](url)